github_projects_board.md

```diff
@@ -0,0 +1,26 @@
+# GitHub Projects board
```

Comment:
@liatsade Some questions:
1. what other columns can be added? for example, I have an "upcoming" column for my project.
2. what columns are mandatory
3. is it true that the AM assigns issues to the developers? Shouldn't the team lead do it?
4. Who does what at each column is not clear - for example, who moves the issue to QA? 
5. Are issues deleted from Merged column?

Suggestion: Show a blank "ideal" board and indicate what is the process of adding/moving issues from them, when things happen and by whom. I think having to do this will force us to decide on a best practices process.

For example (I don't know if what I am writing is true):
"Upcoming" (AM to fill every Thu or as needed per priority) > "Dev Columns" (TL pulls from "Upcoming" every TH and assigns then to the Dev's)>"In Progress" (Dev pulls from their column and puts it here when they are working on the issue)> "Merged" (When ... > ....